<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prototypeFromSamples</class>
 <widget class="QDialog" name="prototypeFromSamples">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Prototypes From Samples</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sample File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sampleFile_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseSampleFile_btn">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select a field for X coordinates:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="xFiled_comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Select a field for Y coordinates:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="yFiled_comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Covariate selection:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addCovariate_btn">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteCovariate_btn">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="covariate_tableWidget"/>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_btn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
